wafer/sponsors/templates/wafer.sponsors/sponsors_block.html

```diff
@@ -6,8 +7,9 @@
             <ul>
                 {% for sponsor in package.sponsors.all %}
                     <li>
-                        {% if sponsor.logo %}
-                            <img src="{{ sponsor.logo.url }}" alt="{{ sponsor.name }}">
+                        {% sponsor_image_url sponsor "logo.png" as logo_url %}
```

Comment:
I don't think it's reasonable to assume it's going to be a png. I'm trying to use SVG where possible.

Shall we just call it `logo` ?